domains/jirafey.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Jirafey",
+        "email": "jirafey@gmail.com"
+    },
+    "record": {
+        "CNAME": "jirafey.github.io"
```

Comment:
Redirect loop, invalid record.

Reply:
i dont understand what needs to be done here, shouldn't it be redirected to is-a.dev?

Reply:
@DEV-DIBSTER